### PR TITLE
Add 'assemble' action_config for Windows.

### LINKED ISSUE #3648: Action_config for 'assemble' missing from Windows crosstool
Reported on [bazel-discuss](https://groups.google.com/d/msgid/bazel-discuss/f0bcc3e2-2e19-4d30-86ab-4eb37a923e1b%40googlegroups.com).

Yun, can cl.exe compile assembly?

### PR DESCRIPTION
**Pull request by sgreenstein**
Allows .asm files to be included in srcs. ml64.exe is used to create .o files which can later be linked. However, this change will not allow custom flags to be passed to ml64.exe other than /I and /D.

Fixes #3648

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by meteorcloudy** [comment]
This is very cool! But it would be greater to add a test for this. Can you add one in https://github.com/bazelbuild/bazel/blob/master/src/test/py/bazel/bazel_windows_test.py ?
I'll later create a `bazel_windows_cpp_test` and move all related tests into that one.

**Comment by sgreenstein** [comment]
@meteorcloudy added a test

File: src/test/py/bazel/bazel_windows_test.py [hunk 1]
```diff
@@ -68,6 +68,48 @@ def testUseMSVCWrapperScript(self):
...
+        '  printf("%d\n", x);',
+        '  return 0;',
+        '}',
+    ])
+
+    exit_code, _, stderr = self.RunBazel(
+        [
+            'build',
+            '//:x', '--verbose_failures',
+            '--cpu=x64_windows_msvc',
```
Comment (meteorcloudy) [inline src/test/py/bazel/bazel_windows_test.py]: We don't need --cpu=x64_windows_msvc anymore, please remove this.

**Comment by sgreenstein** [comment]
Removed --cpu=x64_windows_msvc

**Comment by sgreenstein** [comment]
Friendly ping. Addressed your comments

**Comment by meteorcloudy** [comment]
Oh, so sorry for the delay.

**Comment by meteorcloudy** [comment]
LGTM, I'll import this

**Comment by sgreenstein** [comment]
Fixed some bugs in the test, confirmed it now passes

**Pull request closed without merging**